tests/test_xlson.py

```diff
@@ -122,6 +122,30 @@ def test_choose_image_field(self) -> None:
             },
         )
 
+    def test_required_field(self) -> None:
+        """Test xlson.build_field() - returns a native form choose_image field dict"""
+        options = {
+            "name": "first_name",
+            "label": "What's your name?",
+            "type": "text",
+            "required": "true",
+            "required_message": "Please enter the first name.",
+        }
+        # __import__('ipdb').set_trace()
```

Comment:
Can we please clean up the `ipdb` comment?

Reply:
Apologies. Done